authLdap.php

```diff
@@ -310,6 +326,48 @@ function authLdap_login($user, $username, $password, $already_md5 = false)
 		$loggedInUser = $mapper($loggedInUser);
 	}
 
+	// Set global variable to distinguish LDAP and non-LDAP logins for authorization flow
+	global $authLDAPisLdapLogin;
+	$authLDAPisLdapLogin = true;
+
+	return $loggedInUser;
+}
+
+/**
+ * This method authorizes a user based on their LDAP group memberships
+ * after logging in, including External users if enabled.
+ * 
+ *
+ * @param null|WP_User|WP_Error
+ * @param string $username
+ * @return WP_User object
+ * @conf boolean authLDAP true, if authLDAP should be used, false if not. Defaults to false
+ * @conf boolean authLDAPExternalUsers true, if LDAP should be used for authorization of non-LDAP logins, false if not. Defaults to false
+ * @conf boolean authLDAPLocalWithExternal true, if local users are still allowed when authorizing external users, false if not. Defaults to false
+ * @conf string authLDAPFilter LDAP filter to use to find correct user, defaults to '(uid=%s)'
+ * @conf string authLDAPUidAttr LDAP attribute containing user id (the username we log on with), defaults to 'uid'
+ * @conf string authLDAPDefaultRole default role for authenticated user, defaults to ''
+ * @conf boolean authLDAPGroupEnable true, if we try to map LDAP groups to Wordpress roles
+ * @conf boolean authLDAPGroupOverUser true, if LDAP Groups have precedence over existing user roles
+ */
+function authLdap_authorization($loggedInUser, $username){
+	global $authLDAPisLdapLogin;
+	// Don't trigger if this isn't an LDAP login AND External Users aren't enabled
+	if (!($authLDAPisLdapLogin === true) && !authLdap_get_option('ExternalUsers')) {
+		authLdap_debug('User ineligible for LDAP Authorization ');
+		return $loggedInUser;
+	}
+
+	// If this isn't already an LDAP user, force a reset in the LDAP server list, or LDAP lookups will fail
+	if (!($authLDAPisLdapLogin === true)) {
+		$ldapServerList = authLdap_get_server("reset");
```

Comment:
This should be a boolean `true`, shouldn't it?

Reply:
That or I need to adjust the default value, as mentioned above. Will ponder this a bit.